cypress/integration/version/task_table.ts

```diff
@@ -80,7 +80,7 @@ describe("Task table", () => {
 
   ["NAME", "STATUS", "BASE_STATUS", "VARIANT"].forEach((sortBy) => {
     // TODO: This test doesn't work bc of issues with assertCorrectRequestVariables
```

Comment:
Could you open a ticket for this if we don't already have one? Seems like it's from three years ago back when Spruce was on Cypress 5, and I would guess we might be able to fix it now. Alternatively if it seems unimportant just delete it 🤷

Reply:
Better yet I will include this in the table migration project. https://jira.mongodb.org/browse/EVG-19972